answerking_app/services/order_service.py

```diff
@@ -1,55 +1,48 @@
 from decimal import Decimal, InvalidOperation
 
+from django.db.models import QuerySet
 from django.core.exceptions import ValidationError
 from django.db import DataError
 
 from answerking_app.models.models import Order, Item, Status
-from answerking_app.models.validation.serializers import (
-    OrderSerializer,
-    OrderLineSerializer,
-)
+
 from answerking_app.models.validation.validators import (
     validate_positive_number,
     validate_price,
     validate_address_string,
 )
 
 
-def get_all():
-    orders = Order.objects.all()
-
-    if not orders:
-        return []
-
-    return orders
+def get_all() -> QuerySet[Order]:
+    return Order.objects.all()
 
 
-def get_by_id(order_id):
+def get_by_id(order_id: str) -> Order | None:
     try:
-        order_id = int(order_id)
+        order_id_as_int: int = int(order_id)
     except ValueError:
         return None
 
     try:
-        order = Order.objects.get(pk=order_id)
+        order: Order = Order.objects.get(pk=order_id_as_int)
         return order
     except Order.DoesNotExist:
         return None
 
 
-def create(body):
+def create(body: dict) -> Order | None:
     try:
-        address = validate_address_string(body["address"])
+        address: str = validate_address_string(body["address"])
     except (KeyError, ValidationError):
         return None
 
     status, _ = Status.objects.get_or_create(status="Pending")
 
-    created_order = Order(address=address, status=status, total=0.00)
+    created_order: Order = Order(address=address, status=status, total=0.00)
     created_order.save()
 
     try:
-        order_items = body["order_items"]
+        order_items: dict = body["order_items"]
```

Comment:
I might be wrong, but isn't this a list?

Reply:
You're right. That has been updated

Reply:
I think it's worth defining a TypedDict since it's used often, somethink like we defined here
https://github.com/AnswerConsulting/AnswerKing-Python/blob/develop/answerking_app/tests/API_types.py https://github.com/AnswerConsulting/AnswerKing-Python/blob/develop/answerking_app/views/ErrorType.py